src/app/_components/subscribe-section.tsx

```diff
@@ -3,8 +3,18 @@
 import { useState } from "react";
 
 import { Text } from "./text";
+import { twMerge } from "../_lib/tw-merge";
 
-export function SubscribeSection() {
+const variantToClassNames = {
+  aqua: {
+    buttonClassName: "bg-aqua-100/[.05] text-aqua-100",
+  },
+  teal: {
+    buttonClassName: "bg-teal-100/[.05] text-teal-100",
+  },
```

Comment:
I've seen `aqua-100/[.05]` and `teal-100/[.05]` a few times. Thoughts on making these part of the style?

Reply:
yea at that point it would make sense! I will keep that in mind when refactoring